tests/test_bs.py

```diff
@@ -44,30 +38,42 @@
 
 
 def get_conf(bs_info, client_config, setup_oracle=None, setup_poet=None, args=None):
-
+    """
+    get_conf gather specification information into one ContainerSpec object
+
+    :param bs_info: bootstrap info, type DeploymentInfo
+    :param client_config: client info
+    :param setup_oracle: oracle info
+    :param setup_poet: poet info
+    :param args: arguments for appendage
+    :return: ContainerSpec
+    """
     client_args = {} if 'args' not in client_config else client_config['args']
-
+    # append client arguments
     if args is not None:
         for arg in args:
             client_args[arg] = args[arg]
 
+    # create a new container spec with client configuration
     cspec = ContainerSpec(cname='client', specs=client_config)
 
+    # append oracle configuration
     if setup_oracle:
         client_args['oracle_server'] = 'http://{0}:{1}'.format(setup_oracle, ORACLE_SERVER_PORT)
 
+    # append poet configuration
     if setup_poet:
         client_args['poet_server'] = '{0}:{1}'.format(setup_poet, POET_SERVER_PORT)
 
     cspec.append_args(bootnodes=node_string(bs_info['key'], bs_info['pod_ip'], BOOTSTRAP_PORT, BOOTSTRAP_PORT),
                       genesis_time=GENESIS_TIME.isoformat('T', 'seconds'))
-
+    # append client config to ContainerSpec
     if len(client_args) > 0:
         cspec.append_args(**client_args)
     return cspec
 
 
-def add_multi_clients(deployment_id, container_specs, size=2):
+def add_multi_clients(deployment_id, container_specs, size=2, ret_pods=False):
```

Comment:
add document to these params as well

Reply:
fixed in a8ca3677c850eb09dc69144f4c0ef4fbeef737aa